finalPrep/modZeroHero.js

```diff
@@ -1,84 +1,169 @@
 //   greeting should be assigned to a string that uses concatenation to include the heroName
+var greeting = "Never fear, " + heroName + " is here!";
+console.log(greeting);
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
+var specialAbility = "manifest real objects with code";
+var catchPhrase = "You know, kinda like Green Lantern but without all the green glowing.";
+console.log(`I can ${specialAbility}. ${catchPhrase}`);
 
 // Declare two variables - power AND energy - set to integers
+var power = 15;
+var energy = 26;
 
 // Declare two variables - fullPower AND fullEnergy
 //   fullPower should multiply your current power by 500
+var fullPower = power * 500;
+console.log(fullPower);
 //   fullEnergy should add 150 to your current energy
+var fullEnergy = energy + 150;
+console.log(fullEnergy);
 
 // Declare two variables - isHuman and identityConcealed - assigned to booleans
-
+var isHuman = true;
+var identityConcealed = false;
 
 // Declare two variables - archEnemies AND sidekicks
 //   archEnemies should be an array of at least 3 different enemy strings
 //   sidekicks should be an array of at least 3 different sidekick strings
-
+var archEnemies = ["Distracto", "The Procrastinator", "Debbie Downer"];
+var sidekicks  = ["Challenge Seeker", "Mr. Persistence", "Chronos"]
 // Print the first sidekick to your console
+console.log(sidekicks[0])
 
 // Print the last archEnemy to the console
-
+console.log(archEnemies[2])
 // Write some code to add a new archEnemy to the archEnemies array
+archEnemies.push("Wheel Spinner")
 
 // Print the archEnemies array to console to ensure you added a new archEnemy
-
+console.log(archEnemies)
 // Remove the first sidekick from the sidekicks array
+sidekicks.shift()
 
 // Print the sidekicks array to console to ensure you added a new sidekick
+console.log(sidekicks)
 
 // Create a function called assessSituation that takes three arguments - dangerLevel, saveTheDay, badExcuse
 //   - dangerLevel should be an integer
-//   - saveTheDay should be a string a hero would say once they save the day 
+// - saveTheDay should be a string a hero would say once they save the day
 //   - badExcuse should be a string a hero would say if they are too afraid of the dangerLevel
-
 // Your function should include an if/else statement that meets the following criteria
 //   - Danger levels that are above 50 are too scary for your hero. Any danger level that is above 50 should result in printing the badExcuse to the console
 //   - Anything dangerLevel that is between 10 and 50 should result in printing the saveTheDay string to the console
 //   - If the dangerLevel is below 10, it means it is not worth your time and should result in printing the string "Meh. Hard pass." to the console.
 
-//Test Cases
+function assessSituation(dangerLevel, saveTheDay, badExcuse){
+  if (dangerLevel < 10){
+  console.log("Meh. Hard pass.");
+} else if (dangerLevel <= 50) {
```

Comment:
It is best practice to try to avoid conditions that overlap. For example, a number less than 10 is also less than 50. You can use `&&` to set multiple conditions, or think about having your more complex logic be what lives in your `else` block.